modules/ocm_cluster_rosa_classic/operator_roles/csi_driver_role.tf

```diff
@@ -0,0 +1,35 @@
+resource "aws_iam_role" "csi_drivers_role" {
+  name = "${var.operator_role_prefix}-openshift-cluster-csi-drivers-ebs-cloud-cre"
+  assume_role_policy = jsonencode({
+    Version = "2012-10-17"
+    Statement = [
+      {
+        Action = "sts:AssumeRoleWithWebIdentity"
+        Effect = "Allow"
+        Condition = {
+            StringEquals = {
+                "${var.rh_oidc_provider_url}:sub" = [
+                  "system:serviceaccount:openshift-cluster-csi-drivers:aws-ebs-csi-driver-operator",
+                  "system:serviceaccount:openshift-cluster-csi-drivers:aws-ebs-csi-driver-controller-sa"
+                  ]
+            }
+        }
+        Principal = {
+          Federated = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:oidc-provider/${var.rh_oidc_provider_url}"
+        }
+      },
+    ]
+  })
+
+  tags = {
+    red-hat-managed = true
+    rosa_cluster_id = var.cluster_id
+    operator_namespace = "openshift-cluster-csi-drivers"
+    operator_name = "ebs-cloud-credentials"
+  }
+}
+
+resource "aws_iam_role_policy_attachment" "csi_drivers_role_policy_attachment" {
+  role = aws_iam_role.csi_drivers_role.name
+  policy_arn = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:policy/ManagedOpenShift-openshift-cluster-csi-drivers-ebs-cloud-credent"
```

Comment:
So this might be that eventually, we are actually going over this atm in the CLI as well. Today we use the account roles prefix instead of the operator roles prefix for the operator policy, so hold on to this specific change for a moment

Reply:
@gdbranco good point, so I assume it is supposed to be the account role prefix as of now.

@bardielle the `ManagedOpenShift` is the default option for the account roles prefix, 
but the user can choose another value for this prefix.

Reply:
Changed it to be assigned by a new variable called var.account_role_prefix

Reply:
I found [this function](https://github.com/openshift/rosa/blob/510a8c7e2c25b85a2ec5e30be3c0deeaa5d44ac9/pkg/aws/helpers.go#L369) for calculating the policy name